test/commons/color/get-foreground-color.js

```diff
@@ -1,190 +1,211 @@
-describe('color.getForegroundColor', function () {
-  'use strict';
-
-  var fixture = document.getElementById('fixture');
-  var shadowSupported = axe.testUtils.shadowSupport.v1;
-
-  afterEach(function () {
-    document.getElementById('fixture').innerHTML = '';
+describe('color.getForegroundColor', () => {
+  const { getForegroundColor, Color } = axe.commons.color;
+  const { queryFixture, queryShadowFixture } = axe.testUtils;
+
+  function assertSameColor(actual, expected, margin = 0) {
+    assert.closeTo(actual.red, expected.red, margin);
+    assert.closeTo(actual.green, expected.green, margin);
+    assert.closeTo(actual.blue, expected.blue, margin);
+    // Reduce the margin for alpha, since it is 100-based, rather than 255-based
```

Comment:
On second thought, this should be done differently. the colors are 0 - 255, alpha is 0 - 1. For a fair comparison need to divide this by 255.